build.zig

```diff
@@ -81,7 +81,8 @@ pub fn build(b: *std.Build) void {
     // Tests
 
     var comptime_tests = b.addOptions();
-    const comptime_tests_enabled = b.option(bool, "comptime-tests", "Run comptime tests") orelse true;
+    // TODO: Re-enable comptime tests
+    const comptime_tests_enabled = b.option(bool, "comptime-tests", "Run comptime tests") orelse false;
```

Comment:
Thanks for the TODO!

I fear most of the comptime capabilities in mustache-zig are compromised due to recent changes in how Zig treats comptime memory (only const pointers are allowed to "leak" into runtime code). But let the language stabilize a bit more before trying to make this work again.